finagle-backend/src/main/scala/sttp/client3/finagle/FinagleBackend.scala

```diff
@@ -128,11 +128,7 @@ class FinagleBackend(client: Option[Client] = None) extends SttpBackend[TFuture,
       case MultipartBody(_)       => throw new IllegalArgumentException("Nested multipart bodies are not supported")
     }
 
-    part.fileName match {
-      case Some(_) => FileElement(part.name, ByteArray(content.getBytes: _*), part.contentType, part.fileName)
-      case None    => SimpleElement(part.name, content)
-    }
-
+    FileElement(part.name, ByteArray(content.getBytes: _*), part.contentType, part.fileName)
```

Comment:
I don' think we always want to return a file element?

Reply:
corrected